### PR TITLE
Create cluster-stacks-blogpost

### PR DESCRIPTION
**Pull request by alexander-diab**
Blogpost for Cluster Stacks. English and German below

### CODE REVIEW

File: _i18n/de/_posts/blog/2023-11-27-clusterstacks.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Der Sovereign Cloud Stack hat mit dem Release 5 eine Technical Preview der "Cluster Stacks" veröffentlicht - einem Framework, welches den Produktivbetrieb von Kubernetes-Clustern aus Nutzerperspektive vereinfacht sowie Transparenz bezüglich der Konfiguration und Zusammenstellung der Kernkomponenten der Cluster bietet.
+Damit schafft SCS eine dezentrale und anwenderorientierte Alternative zu den häufig intransparenten und untereinander inkompatiblen kommerziellen "Managed Kubernetes"-Angeboten.
+
+## Kubernetes - schwieriger als gedacht
+Einen Kubernetes-Cluster zu betreiben ist grundsätzlich nicht schwierig. Einfache Kubernetes-Cluster erfüllen allerdings in der Regel nicht die Anforderungen für einen Produktivbetrieb in Bezug auf Stabilität, Sicherheit und Verlustfreiheit. Ein Kubernetes-Cluster in einer Produktionsumgebung muss aufwändig konfiguriert werden, etwa in Bezug auf wichtige Kern-Applikationen und Node-Images, um zum Beispiel gegen Cyber-Angriffe abgesichert zu sein. Fehlendes Wissen führt in der Praxis häufig dazu, dass Unternehmen nicht abgesicherte Cluster für echte Workloads verwenden, welche durch die stark steigende Anzahl von Cyber-Angriffe gefährdet sind.
```
Comment (fkr) [inline _i18n/de/_posts/blog/2023-11-27-clusterstacks.md:19]: Imho I’d prefer it if we don’t speak about “Cyber-Angriffe” - that is snake-oil terminology and imho can be better worded with saying something like ‘und Node-Images, um zum Beispiel den Cluster besser abzusichern’.

**Comment by fkr** [comment]
I miss a reference to the KaaS v1 implementation. In the passage on Cluster-API there could be a short reference to KaaS v1 and that that was already built on top of Cluster-API and that also a migration path is planned.

**Comment by fkr** [comment]
if it is OK (@alexander-diab, @jschoone) I would address the remaining points (adjust the german text to match the english one) and add the reference to KaaS V1 and migration path.

**Comment by alexander-diab** [comment]
> if it is OK (@alexander-diab, @jschoone) I would address the remaining points (adjust the german text to match the english one) and add the reference to KaaS V1 and migration path.

That would be beautiful. Thanks a million

**Comment by alexander-diab** [comment]
Thank you, Felix!

**Review by fkr — APPROVED** [review]

**Pull request merged**